pkgs/dart_model/lib/src/json_buffer/json_buffer_builder.dart

```diff
@@ -297,17 +308,27 @@ class JsonBufferBuilder {
     _buffer.setRange(0, oldBuffer.length, oldBuffer);
   }
 
+  /// Throws if writes are not allowed.
   void _checkAllowWrites() {
     if (!_allowWrites) throw StateError('This JsonBufferBuilder is read-only.');
   }
 
+  /// Explanations of each byte in the buffer.
+  ///
+  /// Run tests with `dart -Ddebug_json_buffer=true test -c source` to enable.
+  ///
+  /// Then, [JsonBufferBuilder#toString] prints additional information for
+  /// each byte.
+  final _Explanations? _explanations =
```

Comment:
This should be static yeah?

Reply:
Class local is a bit neater for the tests, and it doesn't seem to make a difference in the benchmarks, i.e. performance seems to be the same.